# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../perceptualdiff/png.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pdiff/RGBAImage.cpp.vanilla	2006-12-18 12:05:26.000000000 +0000
+++ pdiff/RGBAImage.cpp	2006-12-18 12:06:16.000000000 +0000
@@ -114,7 +114,7 @@
 		}
 	}
 
-	png_read_destroy(png_ptr, info_ptr, end_info);
+	// FIXME: png_read_destroy(png_ptr, info_ptr, end_info);
 	return fimg;
 }
 		   
